### PR TITLE
Init Project

### PR DESCRIPTION
**Pull request by NurAnisAqilah**
ready

### CODE REVIEW

**Comment by NurAnisAqilah** [comment]
ready

**Comment by NurAnisAqilah** [comment]
ready

**Comment by thaodtd-0665** [comment]
commented
Just testing system :p

**Comment by thaodtd-0665** [comment]
ready

File: Gemfile [hunk 1]
```diff
@@ -0,0 +1,54 @@
+source 'https://rubygems.org'
+git_source(:github) { |repo| "https://github.com/#{repo}.git" }
+
+ruby '2.6.5'
+
+# Bundle edge Rails instead: gem 'rails', github: 'rails/rails'
+gem 'rails', '~> 6.0.3', '>= 6.0.3.2'
```
Comment (saobv-2223) [inline Gemfile]: remove all the comments
in ruby files, using " instead of '

File: app/mailers/application_mailer.rb [hunk 2]
```diff
@@ -0,0 +1,4 @@
+class ApplicationMailer < ActionMailer::Base
+  default from: 'from@example.com'
+  layout 'mailer'
```
Comment (saobv-2223) [inline app/mailers/application_mailer.rb]: use " please

File: app/views/layouts/application.html.erb [hunk 3]
```diff
@@ -0,0 +1,15 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <title>SampleApp</title>
+    <%= csrf_meta_tags %>
+    <%= csp_meta_tag %>
+
+    <%= stylesheet_link_tag 'application', media: 'all', 'data-turbolinks-track': 'reload' %>
+    <%= javascript_pack_tag 'application', 'data-turbolinks-track': 'reload' %>
```
Comment (saobv-2223) [inline app/views/layouts/application.html.erb]: use "

File: config/credentials.yml.enc [hunk 4]
```diff
@@ -0,0 +1 @@
+EG7slYG86cPFf9TUEDK2oSMn1djrkhHAEC0H8XmriCbnBIGBBEDdUeRkezzSutBmfjc0PdRdCwq7waqpPvucA/XjaWlClfxvnBpeS4aYRE7WeGJylUDkpmoJ1Nl/UEXmFAsVpaAqFEPT58DwuYoYWtgh3KVKkyQMyfDOP8ZeUOPTzjiGCcpteuoQwP4uX/U8TcivO6dt7OQYAcweh9z0VpWTYynGqOSpZUlqoU43CdnerN7KULy8RpEMPbjKr0E+95qQbMObK4n/k6+JfYzH+qhH8k5ZJwnSKlah1ccJb5Yd4nxqhgveTFSNHvu3fmXOr3nBHIuldwbhcg1VA2N4uHuzVsQ7sD3TGO1Z8c/i+SmZ72yjeKs9vm5c1N2UhsnVGzLccCi3+JwPp7+hupF1soAvsES0aTHrqPWt--+wLtIgFWsx7S75Ti--0+SJkw2DgcBL0iXIQ+/Idg==
```
Comment (saobv-2223) [inline config/credentials.yml.enc:1]: You should create `credentials.yml.enc .example` file and push to git.
Add this file to gitignore and run `git rm --cached config/credentials.yml.enc` to untrack it

File: config/database.yml [hunk 5]
```diff
@@ -0,0 +1,25 @@
+# SQLite. Versions 3.8.0 and up are supported.
+#   gem install sqlite3
+#
```
Comment (saobv-2223) [inline config/database.yml:3]: Do same as credentials.yml.enc file

File: test/application_system_test_case.rb [hunk 6]
```diff
@@ -0,0 +1,5 @@
+require "test_helper"
+
+class ApplicationSystemTestCase < ActionDispatch::SystemTestCase
```
Comment (saobv-2223) [inline test/application_system_test_case.rb:3]: We don't use test folder.
Open `.gitignore` and add `test` to it.
Run `git rm --cached -r test` to untrack it

**Comment by NurAnisAqilah** [comment]
ready

File: Gemfile [hunk 7]
```diff
@@ -0,0 +1,54 @@
...
+
+ruby '2.6.5'
+
+# Bundle edge Rails instead: gem "rails", github: "rails/rails"
+gem "rails", "~> 6.0.3", ">= 6.0.3.2"
+# Use sqlite3 as the database for Active Record
+gem 'sqlite3', '~> 1.4'
+# Use Puma as the app server
+gem 'puma', '~> 4.1'
+# Use SCSS for stylesheets
```
Comment (saobv-2223) [inline Gemfile]: Remove all the comment lines
image

File: app/javascript/packs/application.js [hunk 8]
```diff
@@ -0,0 +1,17 @@
+// This file is automatically compiled by Webpack, along with any other files
+// present in this directory. You're encouraged to place your actual application logic in
+// a relevant structure within app/javascript and only use these pack files to reference
+// that code so it'll be compiled.
+
+require("@rails/ujs").start()
+require("turbolinks").start()
+require("@rails/activestorage").start()
+require("channels")
```
Comment (saobv-2223) [inline app/javascript/packs/application.js:6]: Remove all the comment lines
image

File: config/credentials.yml.enc (hunk 4 above)
Comment (saobv-2223) [inline config/credentials.yml.enc:1]: image

File: config/database.yml (hunk 5 above)
Comment (saobv-2223) [reply config/database.yml:3]: image

File: test/application_system_test_case.rb (hunk 6 above)
Comment (saobv-2223) [reply test/application_system_test_case.rb:3]: image

**Review by saobv-2223 — COMMENTED** [review]
Please, fix all the comments.

**Comment by vanvtt-0952** [comment]
commented

**Comment by NurAnisAqilah** [comment]
ready

**Review by saobv-2223 — COMMENTED** [review]
fix all the comment, please

**Comment by NurAnisAqilah** [comment]
ready

**Comment by NurAnisAqilah** [comment]
ready

**Comment by manhnt-0960** [comment]
merged

**Pull request merged**